pkg/mimirtool/commands/remote_read_test.go

```diff
@@ -150,3 +152,38 @@ func TestTimeSeriesIterator(t *testing.T) {
 	}
 
 }
+
+// See discussion in https://github.com/grafana/mimir/pull/5700 for detailed explanation
+func TestEarlyCommit(t *testing.T) {
+	maxSamplesPerBlock := 1000
+	series := 100
+	samples := 140
+
+	start := int64(1691408300722)
+	inc := int64(60000)
```

Comment:
can this be calculated from constants so it's a bit easier to read? 
```suggestion
	inc := int64(time.Minute / time.Millisecond)
```